### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by vishwajeetgslab**
@JosephPJordan Updated few files text fonts and added FAQ section as subsection for each section. Can you please merge this PR. Thanks

### CODE REVIEW

File: docs/install.rst [hunk 1]
```diff
@@ -158,13 +161,13 @@ Users may click on any of the discovered devices and login to the device. Once l
 
     - In order for the BIG-IP devices to be discovered via LLDP protocol, LLDP needs to be enabled on the BIG-IPs as well as on APIC. To enable LLDP on BIG-IP:
 
-          1. Login to BIG-IP
+      1. Login to BIG-IP
```
Comment (JosephPJordan) [inline docs/install.rst:164]: 1. Log in to the BIG-IP.

File: docs/install.rst [hunk 2]
```diff
@@ -158,13 +161,13 @@ Users may click on any of the discovered devices and login to the device. Once l
...
 
-          2. Click on Interfaces → LLDP → General
+      2. Click on Interfaces → LLDP → General
```
Comment (JosephPJordan) [inline docs/install.rst:166]: 2. Click Interfaces...

File: docs/l2-l3.rst [hunk 3]
```diff
@@ -186,17 +186,21 @@ Create a VLAN for HA setup
 
 4. Select one or more VLANs and move them to the Selected box.
 
-5. The details of all VLANs in the selected box are displayed below. The details consist of the APIC logical interface name.
-After selecting the VLANS click Manage Selected.
+5. The details of all VLANs in the selected box are displayed below. The details consist of the APIC logical interface name.After selecting the VLANS click Manage Selected.
```
Comment (JosephPJordan) [inline docs/l2-l3.rst:189]: ...interface name. After selecting <space after period>

File: docs/l2-l3.rst [hunk 4]
```diff
@@ -186,17 +186,21 @@ Create a VLAN for HA setup
...
 .. note::
     - A VLAN configuration form is displayed to add VLANs and information for one or more self IPs. For VLAN, the fields to be configured are: (1) Name (2) Tag (3) Interface.
     - If a configuration with 2 vCMP guests on separate vCMP hosts in HA configuration has to be supported, the 2 vCMP hosts will need to have the same VLAN name. Hence please add the same VLAN Name for such vCMP Hosts - guests of which hosts are going to form HA pair. The steps to do so are as follows:
-        - Create VLAN on vCMP host 1
-        - Copy VLAN name from host 1 and create VLAN on host 2 with same name
-        - Assign these VLANs to corresponding vCMP Guests vCMP guest 1 (on host 1) and vCMP guest 2 (on host 2)
-        - Create local and floating ips for these VLANs. VLAN and local Self IP should be present on both the BIG-IP devices of the HA pair before Floating Self IPs can be created.
-        - Click Submit.
+    
+      - Create VLAN on vCMP host 1
```
Comment (JosephPJordan) [inline docs/l2-l3.rst:195]: Create a VLAN <and period at the end>

File: docs/l2-l3.rst [hunk 5]
```diff
@@ -186,17 +186,21 @@ Create a VLAN for HA setup
...
+      
+      - Copy VLAN name from host 1 and create VLAN on host 2 with same name
```
Comment (JosephPJordan) [inline docs/l2-l3.rst:197]: Copy the VLAN..  <period at end>

File: docs/l2-l3.rst [hunk 6]
```diff
@@ -186,17 +186,21 @@ Create a VLAN for HA setup
...
+      
+      - Assign these VLANs to corresponding vCMP Guests vCMP guest 1 (on host 1) and vCMP guest 2 (on host 2)
```
Comment (JosephPJordan) [inline docs/l2-l3.rst:199]: .. to the corresponding.. <and period at the end>

File: docs/l2-l3.rst [hunk 7]
```diff
@@ -186,17 +186,21 @@ Create a VLAN for HA setup
...
+      
+      - Create local and floating ips for these VLANs. VLAN and local Self IP should be present on both the BIG-IP devices of the HA pair before Floating Self IPs can be created.
```
Comment (JosephPJordan) [inline docs/l2-l3.rst:201]: ..floating IPs...
The VLAN and local Self

File: docs/l2-l3.rst [hunk 8]
```diff
@@ -212,11 +216,9 @@ Create Self IPs
 ```````````````````````````
 
 1. Click the L2-L3 Network Management tab.
+2. Select one or more available vlans and move them to the Selected box. These VLANs are already present on the vCMP Guest BIG-IP device After selecting the VLANS click Manage Selected.
```
Comment (JosephPJordan) [inline docs/l2-l3.rst:219]: ..available VLANs
<seems like you are missing a period after Guest BIG-IP device.. either that or comma and no cap 'after'>

File: docs/l4-l7.rst [hunk 9]
```diff
@@ -293,31 +288,31 @@ Upload a new FAST Template set using Basic mode (Application subtab)
...
+---------------------------------------------------------------------
 
 1. Select the Basic mode on Application subtab.
 
 2. From the Partition list, click :guilabel:`Create New Partition`.
 
 3. From the :guilabel:`Template` list, click the template which is to be deleted.
 
-3. Click the Delete icon besides template name.
+4. Click the Delete icon besides template name.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:313]: either "beside the template name" or "next to the template name"

File: docs/l4-l7.rst [hunk 10]
```diff
@@ -293,31 +288,31 @@ Upload a new FAST Template set using Basic mode (Application subtab)
...
 
-4. The template will get deleted from the BIG-IP FAST Database.
+5. The template will get deleted from the BIG-IP FAST Database.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:315]: The template will be deleted from...

File: docs/l4-l7.rst [hunk 11]
```diff
@@ -327,29 +322,34 @@ Delete custom FAST Template Set using Basic mode (Application subtab)
...
 
 1. Click the Application Inventory tab on the L4-L7 App Services tab.
 
 2. This will display a list of all the AS3 applications present on the BIG-IP device. The list will have the following columns:
 
-Partition   
-            - Displays BIG-IP partition of the application
+   **Partition**   
+   
+   - Displays BIG-IP partition of the application
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:333]: Displays the BIG-IP..

File: docs/l4-l7.rst [hunk 12]
```diff
@@ -327,29 +322,34 @@ Delete custom FAST Template Set using Basic mode (Application subtab)
...
 
-Application 
-            - Displays application name. Each application name in this table has a hyperlink to the Application sub-tab to respective mode (either Basic/Advanced) used during creation of the application.
+   **Application** 
+    
+   - Displays application name. Each application name in this table has a hyperlink to the Application sub-tab to respective mode (either Basic/Advanced) used during creation of the application.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:337]: Displays the application name.

File: docs/navigate.rst [hunk 13]
```diff
@@ -143,31 +143,29 @@ View VIP Dashboard
...
+   b. **View Stats** - The dashboard displays a **View Stats** link beside the default pool. It displays the total pool member count and the availability and enabled state of the pool  members using a pie chart.
          
-         c. **View Logs** - The dashboard displays a **View Logs** link beside the VIP, which, when clicked opens a popup window and displays all the logs for the selected Virtual Server. 
+   c. **View Logs** - The dashboard displays a **View Logs** link beside the VIP, which, when clicked opens a popup window and displays all the logs for the selected Virtual Server. 
          
-            .. note::
+      .. note::
 
-               - It is possible to also view all logs instead of the filtered logs.
+        - It is possible to also view all logs instead of the filtered logs.
```
Comment (JosephPJordan) [inline docs/navigate.rst:160]: It is also possible to view all logs...

File: docs/navigate.rst [hunk 14]
```diff
@@ -181,31 +179,29 @@ View Node Dashboard
...
 5. The Visibility Dashboard displays the information for the selected VIP including Node name, route domain and monitor.
 
-         a. **View Logs** - The dashboard displays a **View Logs** link beside the Node, which, when clicked opens a popup and all the logs for the selected Node.
+   a. **View Logs** - The dashboard displays a **View Logs** link beside the Node, which, when clicked opens a popup and all the logs for the selected Node.
 
-            .. note::
+      .. note::
 
-               - It is possible to also view all logs instead of the filtered logs.
+         - It is possible to also view all logs instead of the filtered logs.
```
Comment (JosephPJordan) [inline docs/navigate.rst:192]: It is also possible to view all...

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
Hi @vishwajeetgslab, i know most of the content that was changed isn't necessarily new changes, and perhaps I missed it before, but a few minor changes in this batch.

**Comment by vishwajeetgslab** [comment]
@JosephPJordan Thanks for review. I have made the changes as per review comments.

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**